app/assets/javascripts/discourse/components/choose-topic.js.es6

```diff
@@ -24,6 +48,17 @@ export default Component.extend({
     this.search(this.topicTitle);
   },
 
+  @observes("selectedTopic")
```

Comment:
We are trying to avoid observers wherever possible. Could you instead have the action that selects a topic trigger this operation rather than observing?